scripts/check-docs.sh

```diff
@@ -10,38 +8,39 @@ shopt -s globstar
 #   Checks that the contents of /docs folder are valid. This includes:
 #
 #     1. Ensuring the the .cue files can compile.
-#     2. Link validation.
+#     2. In CI, ensuring the the .cue files are properly formatted.
 
-DOCS_PATH="docs"
+cd "$(dirname "${BASH_SOURCE[0]}")/.."
 
-if ! [ -x "$(command -v cue)" ]; then
-  echo 'Error: cue is not installed.' >&2
+read-all-docs() {
+  scripts/cue.sh list | sort | xargs cat -A
+}
+
+if ! cue version >/dev/null; then
+  echo 'Error: cue is not installed'
   exit 1
 fi
 
-if [[ -z "${CI:-}" ]]; then
-  echo "Skipping local formatting - reserved for CI"
+if [[ "${CI:-"false"}" != "true" ]]; then
+  echo "Skipping cue files format validation - reserved for CI"
 else
-  echo "Validating ${DOCS_PATH}/**/*.cue formatting."
+  echo "Validating cue files formatting..."
 
-  cue fmt ${DOCS_PATH}/**/*.cue
-  status="$(git status --porcelain ${DOCS_PATH})"
+  STATE_BEFORE="$(read-all-docs)"
+  scripts/cue.sh fmt
+  STATE_AFTER="$(read-all-docs)"
 
-  [[ -z "$status" ]] || {
-    echo >&2 "Incorrectly formatted Cue files"
-    echo >&2 "$status"
-    git diff ${DOCS_PATH}
+  if [[ "$STATE_BEFORE" != "$STATE_AFTER" ]]; then
+    echo "Incorrectly formatted CUE files"
```

Comment:
Could we output `diff --unified <(echo $STATE_BEFORE) <(echo $STATE_AFTER)` here? I did appreciate seeing the diffs in CI.